### PR TITLE
Quickstart for Nginx ingress controller

### PR DESCRIPTION
**Pull request by thomasvn**

### CODE REVIEW

**Comment by jessegoodier** [comment]
I prefer not kube-system. This is what I use, single line:
```
helm install ingress-nginx ingress-nginx \
  --repo https://kubernetes.github.io/ingress-nginx \
  --namespace ingress-nginx
```

File: ingress-examples/README.md [hunk 1]
```diff
@@ -14,5 +14,20 @@
 
 [ingress-subdir-non-root.yaml](ingress-subdir-non-root.yaml): use your.com/kubecost instead of kubecost on the root path
 
+## Quickstart if using Nginx Ingress Controller (`ingressClassName: nginx`)
```
Comment (linhlam-kc) [inline ingress-examples/README.md:18]: I recommend to add a few lines here to elaborate more about the Nginx ingress controller and user can find more details at Nginx doc https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/

File: ingress-examples/README.md [hunk 2]
```diff
@@ -14,5 +14,20 @@
...
 
+```sh
+# 1. Nginx helm chart
+$ helm repo add nginx-stable https://helm.nginx.com/stable
+$ helm repo update
+$ helm install nginx nginx-stable/nginx-ingress -n kube-system
```
Comment (linhlam-kc) [inline ingress-examples/README.md:23]: it should be installed in a separate namespace than `kube-system` . Maybe something like this:

```bash
$ helm repo add nginx-stable https://helm.nginx.com/stable
$ helm repo update
$ export NAMESPACE="nginx-ingress" # update with your desired namespace
$ helm install nginx nginx-stable/nginx-ingress -n $NAMESPACE --create-namespace
```

**Review by linhlam-kc — CHANGES_REQUESTED** [review]

File: ingress-examples/README.md (hunk 2 above)
Comment (jessegoodier) [reply ingress-examples/README.md:23]: nginx doesn't maintain ingress-nginx. This is the docs for ingress-nginx https://kubernetes.github.io/ingress-nginx/

which is very confusing. 

NGINX maintains nginx-ingress and nginx-plus, which we don't use.

File: ingress-examples/README.md (hunk 1 above)
Comment (thomasvn) [reply ingress-examples/README.md:18]: @linhlam-kc Will do!

File: ingress-examples/README.md (hunk 2 above)
Comment (thomasvn) [reply ingress-examples/README.md:23]: Agree with both that the ingress controller should be in a separate namespace.

I'll update the docs with this command:

```
helm install ingress-nginx ingress-nginx \
  --repo https://kubernetes.github.io/ingress-nginx \
  --namespace ingress-nginx
```

And this doc:

https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/

**Pull request merged**